create_documentation_pdf.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker run --rm -it -v $(pwd):/pandoc geometalab/pandoc bash -c "pandoc -t html5 documentation.md -o documentation.html; wkhtmltopdf --encoding 'utf-8' toc documentation.html documentation.pdf"
```

Comment:
I guess if one of these inside-the-container commands fails, it doesn't make sense to execute the rest of them. So either chain them with `&&` instead of `;` or prepend `set -e;`.